### PR TITLE
Apply Media queries to my project

### PR DESCRIPTION
**Pull request by Alfredbis29**
**PORTFOLIO: DESKTOP-VERSION**

 **Description**
- [ ] For the fourth project in my portfolio website, I will:

- I use media queries to make sure that my website displays correctly on desktop screens.
- I am going to apply Media Queries to responsiveness so that my portfolio will read any screen size.

**Project requirements**

- Make sure that my desktop version looks like the one in the Figma design template that I have chosen.
- assume that the desktop width should be in 992px - 1100px range.
- I use Grid and Flexbox in order to layout the elements on the page.

**I made all requirements from the previous review!!!!**

### CODE REVIEW

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Alfredbis29, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style/style.css [hunk 1]
```diff
@@ -775,7 +775,7 @@ li {
   color: rgb(255, 255, 255);
 }
 
-/* Media queries */
+/* here i used media queries */
```
[the remark this answers is not in the document]
Comment (Alfredbis29) [reply style/style.css]: IS not empty is the one which contain all my work section

**Review by thecodechaser — APPROVED** [review]
## Hi @Alfredbis29 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a title and summary. ✔️ 
- Good commit messages. ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alfredbis29** [comment]
> ## Hi @Alfredbis29 👋,
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉
> 
> ## To Highlight 🎉
> * No linter errors. ✔️
> * GitHub flow is followed. ✔️
> * Professional README file. ✔️
> * PR has a title and summary. ✔️
> * Good commit messages. ✔️
> 
> ## Optional suggestions
> * [ ]  To make your README file more professional, You can also add screenshots of the desktop design.
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thanks so much for the code review

**Comment by Alfredbis29** [comment]
Hello Ranjeet,

I hope you are doing well, thank you for the code review.
It was somehow hard but I made it.

thank you so much

Best,
Alfred Bisimwa

**Comment by thecodechaser** [comment]
@Alfredbis29 Your welcome😄

**Pull request merged**